src/energyplus/ForwardTranslator.cpp

```diff
@@ -1806,6 +1806,16 @@ namespace energyplus {
         retVal = translateHeatExchangerAirToAirSensibleAndLatent(mo);
         break;
       }
+      case openstudio::IddObjectType::OS_HeatExchanger_Desiccant_BalancedFlow: {
+        model::HeatExchangerDesiccantBalancedFlow mo = modelObject.cast<HeatExchangerDesiccantBalancedFlow>();
+        retVal = translateHeatExchangerDesiccantBalancedFlow(mo);
+        break;
+      }
+      case openstudio::IddObjectType::OS_HeatExchanger_Desiccant_BalancedFlow_PerformanceDataType1: {
+        model::HeatExchangerDesiccantBalancedFlowPerformanceDataType1 mo = modelObject.cast<HeatExchangerDesiccantBalancedFlowPerformanceDataType1>();
+        retVal = translateHeatExchangerDesiccantBalancedFlowPerformanceDataType1(mo);
+        break;
+      }
```

Comment:
Ok, not added to the list of objects to always translate, loop will be responsible to trigger HX, and HX will trigger PerfData. LGTM

Reply:
(well, probably not loop, but AirLoopHVACOutdoorAirSystem)